unstract/connectors/src/unstract/connectors/filesystems/minio/minio.py

```
@@ -12,8 +13,55 @@
+        async def _probe(name: str) -> bool:
+            async with sem:
+                return await self._is_bucket_accessible(name)
+
+        results = await asyncio.gather(*(_probe(b["name"]) for b in buckets))
+        return [b for b, ok in zip(buckets, results, strict=True) if ok]
+
+    async def _is_bucket_accessible(self, name: str) -> bool:
+        try:
+            await self._call_s3("list_objects_v2", Bucket=name, MaxKeys=1)
+            return True
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)